explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -976,6 +976,7 @@ export const buildExportMethodManifestDownload = (
       "Request a file manifest for the current query containing the full list of selected files and the metadata for each file.",
     route: ROUTE_MANIFEST_DOWNLOAD,
     title: "Download a File Manifest with Metadata for the Selected Data",
+    trackingId: "button_request_file_manifest",
```

Comment:
Would you be able to update this to kebab case?

Reply:
Whoops, so sorry! Just fixed it @MillenniumFalconMechanic!